config/asg-cfg.example

```diff
@@ -0,0 +1,7 @@
+{
+    "activities": "ActivitiesBoss-Activities-ABC123",
```

Comment:
If the users `521.boss` or `production.boss` value is passed in on the command line, these values can be created by `getattr(lib.names.AWSNames(), 'activities')` (example)

Reply:
Coming back to building multiple boss stacks. Wouldn't this create conflict if the user has multiple ActivitiesBoss-Activities?

Reply:
Regarding why not all ASGs are supported:
Currently if auth is shut down all the data in it would be lost unless we made use of an RDS to make sure we could save the data. That's what we currently have set up in the smaller bosslet, and in that case the auth instance could be stopped. In this case, do you have any recommendations as to how to sop auth?

Reply:
Multiple boss stacks. So each stack can only have one activities ASG, so no concert with looking them up. This does bring to mind that unless the save files are unique that trying to turn off multiple boss stacks can mean losing state.

It would be helpful to document why not all of the ASGs are supported so that someone doesn't accidently add one to the config file that would cause an issue.

Supporting the Auth ASG could be done using the same check as the core config uses to see if the auth RDS should be created. If there is an auth db then you should be able to shut the ec2 instances down. You don't need to add it right now if not needed.

Reply:
Sure thing I'll document it at the top of the script.